hydra-node/test/Hydra/ModelSpec.hs

```diff
@@ -217,38 +217,20 @@ prop_checkHeadOpensIfAllPartiesCommit =
 
 headOpensIfAllPartiesCommit :: DL WorldState ()
 headOpensIfAllPartiesCommit = do
-  seedTheWorld
-  initHead
+  _ <- seedTheWorld
+  _ <- initHead
   everybodyCommit
-  observeHeadOpened
+  void $ eventually' ObserveHeadIsOpen
```

Comment:
Could: use `action_` in this eventually, initHead and seedTheWorld to avoid the `void`s